### DISCUSSION TITLE
Authentication to change the password

### DISCUSSION THREAD

**Discussion by kumawatanupriya** [Project-Books/book-project#681] [Q&A] [answered]
Hi @knjk04 , In this issue https://github.com/Project-Books/book-project/issues/232

We need to change the password for a particular user, 
In new api,  how would I know which user is logged in(for whom I have to change the password) , I don't see any authentication in code.

**Comment by knjk04** [comment, marked as answer]
Hi, 

Try using `getCurrentUser()` in UserService.java

https://github.com/Project-Books/book-project/blob/0.2.0/backend/src/main/java/com/karankumar/bookproject/backend/service/UserService.java#L85

**Reply by kumawatanupriya** [reply]
Thanks @knjk04
